packages/lib/src/io-command.ts

```diff
@@ -363,6 +402,11 @@ export abstract class Listing<L> extends Outputable {
 	}
 }
 
+/**
+ * Extend this class for your command if you need to accept input but the
+ * API call you're making doesn't have complex output. There are currently
+ * no examples of this in the CLI (as of August 2020).
+ */
```

Comment:
Should we have it then? Maybe we can clean up in support of [YAGNI](https://martinfowler.com/bliki/Yagni.html).

Reply:
If this were in the packages/cli, I would have already removed it but since it's in the packages/lib for plugins to use, I think we should err on the side of being more complete.